### DISCUSSION TITLE
Expand the preview window

### DISCUSSION THREAD

**Discussion by Shamaoke** [junegunn/fzf#3453] [Ideas]
Hi.

Fzf allows you to toggle display of the preview window. This is achieved by using the `toggle-preview` action. In addition to this functionality, Fzf also allows you to toggle expansion of this window. This is achieved by using the `change-preview-window(...)` action. E. g.

````
find $PWD -type f | \
  fzf --preview='cat {1}' \
      --preview-window='right,50%' \
      --bind='alt-e:change-preview-window(right,99%|right,50%)'
````

The maximum amount by which the program allows you to expand the window is 99%. In this case a small piece of the selection list will remain on the edge. It doesn't look very neat, and besides, the remaining tiny part of the selection list doesn't carry any useful information.

What is the reason of the 99% maximum preview window size?

Is it possible to expand the preview window completely to cover the entire Fzf window, hiding the selection list?

Thanks.

**Comment by t3e** [comment]
I agree 100%. I think the reasoning behind 99% is that you can still see which entry is selected. But i would very much prefer to at least have the option to have 100% height (or with). I think there should be no sacrifice doing this, but maybe I'm wrong?
